Gemfile

```diff
@@ -1,5 +1,3 @@
 source 'https://rubygems.org'
 
 gem 'github-pages', group: :jekyll_plugins
-gem 'json'
-gem 'rb-inotify'
```

Comment:
I've added these dependencies, because setting up a local GitHub Pages/Jekyll installation did not work without them. Apparently, there were some issues with the dependencies of the `github-pages` Gem. These issues seem to have gone now. At least I didn't need the dependencies when setting up GitHub Pages/Jekyll on my new laptop.